### ISSUE TITLE
[DevResearch] DOM Slider / Toggle when collapse

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2943]
**User Story:**
As Spark we want to look into dom-slider. 

**CURRENT STATE:**
DOM-slider-hidden is not hiding the content from screen readers. This is only evident after showing and then hiding again. 
- a.) Try to fix it with configuration. From Rob: "I think passing in 'none' as the last argument there might fix it" https://github.com/BrentonCozby/dom-slider/blob/master/src/dom-slider.js#L206
- b.) If that doesn't work, submit a PR to dom-slider
- c.) If that doesn't work, drop our dependency on dom-slider and write it ourselves
- d.) If that's too hard, fork dom-slider and fix it ourselves like we did with tiny-date-picker
4. Include a writeup of the no-JS support in the docs. If toggle.js is not available, HTML Toggle contents are always visible.

**AC:**
- Research dom-slider 
- work with PO to create follow up issues to implement.
